Tests/TensorUtilsTests/LogitsWarperTests.swift

```diff
@@ -87,6 +87,36 @@ final class LogitsWarperTests: XCTestCase {
         XCTAssertEqual(result5.logits, [2, 1, 0], accuracy: accuracy)
     }
 
+    func testRepetitionPenaltyWarper() {
```

Comment:
Thanks for adding tests! Did you verify results against the reference transformers implementation?